config/development.js

```diff
@@ -25,7 +26,7 @@ module.exports = {
   },
 
   cors: {
-    allowedOrigins: /(\.realdevsquad\.com$)|(localhost)/, // Allow *.realdevsquad.com and localhost for non-production envs
+    allowedOrigins: /(https:\/\/(.*\.)*realdevsquad\.com$)|(localhost)/, // Allow *.realdevsquad.com and localhost for non-production envs
```

Comment:
currently this will allow a.b.realdevsquad.com, that should not happen I guess please check and fix

Reply:
fixed it